src/rejection_intake.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+if (!defined('not_direct_access') && !empty($_POST['pre']) && !empty($_POST['post']) && !empty($_REQUEST['hunter_id_hash'])) {
```

Comment:
you don't need to check for missing not_direct_access here. We use that to check that the required files are not accessed directly. the intake should be missing it, so it's ok not to check that it's missing.